intercom_flutter/lib/intercom_flutter.dart

```diff
@@ -2,65 +2,45 @@ library intercom_flutter;
 
 import 'dart:async';
 
-import 'package:flutter/services.dart';
+import 'package:intercom_flutter_platform_interface/intercom_flutter_platform_interface.dart';
 
-enum IntercomVisibility { gone, visible }
+/// export the [IntercomVisibility] enum
+export 'package:intercom_flutter_platform_interface/intercom_flutter_platform_interface.dart'
+    show IntercomVisibility;
```

Comment:
Is this needed?

Reply:
yes, otherwise you will not be able to use IntercomVisibility enum in the apps where you use this package intercom_flutter.

Reply:
as IntercomVisibility enum is available in the intercom_flutter_platform_interface